script.js

```diff
@@ -0,0 +1,188 @@
+/*var reset = document.getElementById("reset");
+function turnPageRed() {
+  document.body.style.backgroundColor = "red";
+}
+reset.addEventListener("click", turnPageRed);*/
+var submitTally = 0;
+var userWonGameReset = 0;
+
+function xAndBlank(temporaryPlaceholderVariable, quadrant, actualTextSpace) {
```

Comment:
This function's name isn't very clear to me, I'm having trouble understanding what it does. Try to give functions names that say what they do.